src/System.IdentityModel.Tokens.Jwt/JwtSecurityTokenHandler.cs

```diff
@@ -36,6 +36,8 @@ public class JwtSecurityTokenHandler : SecurityTokenHandler
         private static string _shortClaimType = _namespace + "/ShortTypeName";
         private bool _mapInboundClaims = DefaultMapInboundClaims;
 
+        internal ITelemetryInstrumentation _telemetryClient = new ConfigurationManagerTelemetryInstrumentation();
```

Comment:
This looks strange. Can we put all telemetry methods under a common interface / default implementation that is not class-dependent?

Reply:
Good call. Renamed to `TelemetryInstrumentation`